### PR TITLE
Documentation added for CES (SDK + OSC), Identity Credential (OSC) and rework of README.rst

### PR DESCRIPTION
**Pull request by tischrei**

### CODE REVIEW

File: doc/source/sdk/resources/index.rst [hunk 1]
```diff
@@ -10,6 +10,7 @@ Open Telekom Cloud Resources
    Anti DDoS Service (Anti-DDoS) <anti_ddos/index>
    AutoScaling Service (AS) <auto_scaling/index>
    Cloud Container Engine (CCE) <cce/index>
+   Cloud Cloud Eye Service (CES) <ces/index>
```
Comment (lego963) [inline doc/source/sdk/resources/index.rst]: Cloud Cloud

File: doc/source/sdk/proxies/index.rst [hunk 2]
```diff
@@ -8,6 +8,7 @@ Service Proxies
    AutoScaling Service (AS) <auto_scaling>
    Cloud Container Engine v1 (CCEv1) <cce_v1>
    Cloud Container Engine v2 (CCE) <cce_v3>
+   Cloud Eye (CES) <ces>
```
Comment (lego963) [inline doc/source/sdk/proxies/index.rst]: Cloud Eye Service

**Review by anton-sidelnikov — APPROVED** [review]

**Comment by gtema** [comment]
regate

**Comment by gtema** [comment]
regate

**Pull request merged**